x-pack/plugins/ml/common/constants/messages.test.ts

```diff
@@ -135,13 +137,15 @@ describe('Constants: Messages parseMessages()', () => {
       {
         id: 'success_influencers',
         status: 'success',
-        text: 'Influencer configuration passed the validation checks.',
+        heading: 'Influencer configuration',
+        text: 'Passed the validation checks.',
         url: 'https://www.elastic.co/guide/en/machine-learning/mocked-test-branch/ml-ad-run-jobs.html#ml-ad-influencers',
       },
       {
         id: 'half_estimated_mml_greater_than_mml',
         mml: '1MB',
         status: 'warning',
+        heading: 'Model memory limit',
         text: 'The specified model memory limit is less than half of the estimated model memory limit and will likely hit the hard limit.',
         url: 'https://www.elastic.co/guide/en/machine-learning/mocked-test-branch/ml-ad-run-jobs.html#ml-ad-model-memory-limits',
       },
```

Comment:
Nit - should the two objects below here need `heading`s too? Also `skipped_extended_tests` near the start of the file?